src/DssSpell.t.base.sol

```diff
@@ -1,5 +1,18 @@
-// SPDX-FileCopyrightText: © 2021-2022 Dai Foundation <www.daifoundation.org>
+// SPDX-FileCopyrightText: © 2021 Dai Foundation <www.daifoundation.org>
```

Comment:
Did we want to remove the `2022` on these license headers?

Reply:
Our latest standard on this says to leave just the file creation year.
see: https://hackmd.io/@naszam/S1Sbl5m55